### PR TITLE
Fixed Broken links

### PR DESCRIPTION
**Pull request by dhanushuUzumaki**

### CODE REVIEW

File: bedrock/foundation/templates/foundation/trademarks/policy.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h1 class="page-title">{{ _('Mozilla Trademark Policy') }}</h1>
 
   <p>{% trans %}When it comes to the Mozilla Marks, there are some cool things you can do and some cool things you can't do - at least not without asking Mozilla.{% endtrans %}</p>
 
-  <p>{% trans store="//store.mozilla.org", comm="//communitystore.mozilla.org/" %}
+  <p>{% trans store="https://store.mozilla.org", comm="https://communitystore.mozilla.org/" %}
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/policy.html]: These two links should be `http`.

**Comment by alexgibson** [comment]
Can you please include the bug number in your commit message? Please see the docs for guidance: http://bedrock.readthedocs.org/en/latest/contribute.html#git-commit-messages

Thanks!

**Comment by dhanushuUzumaki** [comment]
Commit message changed and one more link fixed

File: bedrock/foundation/templates/foundation/trademarks/policy.html [hunk 2]
```diff
@@ -122,7 +122,7 @@ <h1 class="page-title">{{ _('Mozilla Trademark Policy') }}</h1>
 
   <p>{% trans %}When it comes to the Mozilla Marks, there are some cool things you can do and some cool things you can't do - at least not without asking Mozilla.{% endtrans %}</p>
 
-  <p>{% trans store="//store.mozilla.org", comm="//communitystore.mozilla.org/" %}
+  <p>{% trans store="https:http//store.mozilla.org", comm="http://communitystore.mozilla.org/" %}
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/policy.html]: You have a typo here

File: bedrock/foundation/templates/foundation/trademarks/policy.html [hunk 3]
```diff
@@ -122,7 +122,7 @@ <h1 class="page-title">{{ _('Mozilla Trademark Policy') }}</h1>
 
   <p>{% trans %}When it comes to the Mozilla Marks, there are some cool things you can do and some cool things you can't do - at least not without asking Mozilla.{% endtrans %}</p>
 
-  <p>{% trans store="//store.mozilla.org", comm="//communitystore.mozilla.org/" %}
+  <p>{% trans store="http//store.mozilla.org", comm="http://communitystore.mozilla.org/" %}
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/policy.html]: Missing colon `:` (please test your changes locally when making fixes, thanks!)

Comment (dhanushuUzumaki) [inline bedrock/foundation/templates/foundation/trademarks/policy.html]: Sorry!
Will not happen again.

**Comment by alexgibson** [comment]
Looks good, thanks! I'll take a look shortly and squash your commits before merging. Thanks for the PR @dhanushuUzumaki :cake:

**Comment by alexgibson** [comment]
I squashed your four commits into one and reworded your commit message to `[fix bug 1235039] Broken links on Trademark Policy page`. Merged in 026f376f02abcd98668ee7da2ed418abe32c993c

Thanks again, @dhanushuUzumaki!

**Pull request closed without merging**